qiskit_experiments/tomography/basis/fitter_basis.py

```diff
@@ -0,0 +1,106 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Fitter basis classes for tomography analysis.
+"""
+from typing import Iterable, Optional, Dict
+import numpy as np
+from .base_basis import BaseFitterMeasurementBasis, BaseFitterPreparationBasis
+
+
+class FitterMeasurementBasis(BaseFitterMeasurementBasis):
+    """Measurement basis for tomography fitters"""
+
+    def __init__(
+        self,
+        povms: Iterable[Dict[int, np.ndarray]],
```

Comment:
Can this be `Iterable[Tuple[str, Dict[int, np.ndarray]]` to keep the label of basis?

Reply:
I would rather not.

Reply:
Okey, now the class docstring has nice table. Perhaps this should be enough.